backend/btrixcloud/models.py

```diff
@@ -1919,6 +1924,9 @@ class OrgMetrics(BaseModel):
     pageCount: int
     crawlPageCount: int
     uploadPageCount: int
+    uniquePageCount: int
+    crawlUniquePageCount: int
```

Comment:
Shouldn't crawls always have unique pages?

Reply:
ah nevermind, that's on the org!

Reply:
yes and also uploads can have multiple captures of the same URL so can have different numbers within an archived item as well